### PR TITLE
[WOE] Implement Stockpiling Celebrant

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Stockpiling Celebrant]]

File: Mage.Sets/src/mage/cards/s/StockpilingCelebrant.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        this.subtype.add(SubType.DWARF);
+        this.subtype.add(SubType.KNIGHT);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // When Stockpiling Celebrant enters the battlefield, you may return another target nonland permanent you control to its owner's hand. If you do, scry 2.
+        this.addAbility(new EntersBattlefieldTriggeredAbility(new DoIfCostPaid(
+                new ScryEffect(2, false),
+                new ReturnToHandChosenControlledPermanentCost(new TargetControlledPermanent(filter))
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/s/StockpilingCelebrant.java:43]: this needs to target when it triggers and not when it resolves, you can just have `ReturnToHandTargetEffect` and `ScryEffect` and have the ability be optional

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/s/StockpilingCelebrant.java (hunk 1 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/s/StockpilingCelebrant.java:43]: Uhm that would not work, as if you have no permanent to return, you should have the option to scry.
Will definitively rethink the implementation.

Comment (Susucre) [reply Mage.Sets/src/mage/cards/s/StockpilingCelebrant.java:43]: Oh nevermind the spell would fizzle without targets.

**Pull request merged**